code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -2793,6 +2793,20 @@ datum
 			transparency = 255
 			taste = "pungent"
 
+		fooddrink/soysauce
+			name = "soy sauce"
+			id = "soysauce"
+			description = "A dark brown sauce brewed from soybeans and wheat. Salty!"
+			reagent_state = LIQUID
+			fluid_r = 53
+			fluid_g = 33
+			fluid_b = 28
+			transparency = 255
+
+			on_mob_life(var/mob/M, var/mult = 1)
+				M.reagents.add_reagent("salt", 0.5 * mult)
+				..()
+
```

Comment:
should add a brew_result to soybeans for this chem

Reply:
> should add a brew_result to soybeans for this chem

Without creating an entirely new mechanic for condiment production, soy sauce reagent can't be converted into soy sauce bottles. I don't see a point to this.

Reply:
why not make them refil soy sauce bottles?

Reply:
> why not make them refil soy sauce bottles?

all condiments are consumables, they can't be refilled. this mechanic doesn't exist for any other condiment in the game

Reply:
marked as resolved. giving a brew_result to soybeans will yield a practically useless chem that can't even be used for the purposes that soy sauce was added for.